project-templates/ecs/basic/README.md

```diff
@@ -17,18 +17,17 @@ Pre-Requisites:
 
 ### Terraform Cloud Workspaces
 
-1. (CLI-driven only) Create terraform cloud [workspaces](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/creating) in the TFC User Interface based on the following terraform configurations provided in this example:
+1. Create a [global variable set](https://developer.hashicorp.com/terraform/tutorials/cloud-get-started/cloud-workspace-create#create-the-workspace) for your AWS Credentials.
+
+2. Create terraform cloud [workspaces](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/creating) in the TFC User Interface based on the following terraform configurations provided in this example:
```

Comment:
Something I wanted to bring up is that "create workspaces based on these configurations" is not clear to me. Should I do `terraform init` in a clone/fork of this repo?

Reply:
Yeah that wasn’t super clear to me either. I also got the names of the workspaces wrong on the first try.

Reply:
Thank you for this change!